### ISSUE TITLE
SD prefix/abbreviation. What does it stand for?

### ISSUE THREAD

**Issue by Hoord** [SDWebImage/SDWebImage#3492]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### SD = ?
Hi, sorry to write this in the issues, but I can't help but wonder what the SD prefix in SDWebImage means? There is no information about it anywhere on the Internet. I would love to know about its origin and its meaning.

Thanks!

**Comment by dreampiggy** [comment]
**Simple Design (SD)**, I guess.

The initial commit: https://github.com/SDWebImage/SDWebImage/commit/9cc8f904ab081f5d5a1a6e97ab3aa658a28dc3b5
shows that Olivier Poitrey named it into`DailyMotionWebImage` because he is in that company [Daily Motion](https://en.wikipedia.org/wiki/Dailymotion).

And then, in the 1.0.0 version, he rename this library into `SD` team name (not company name), see: https://github.com/SDWebImage/SDWebImage/commit/a7734af5116af832781ba043993e060141f837ca

**Comment by cherishloveyou** [comment]
CocoaPods versrion update

**Comment by dreampiggy** [comment]
> CocoaPods version update,please!

I can not understand what's your reply for. If you're facing new issue, search the history issue or create new GitHun issue instead.

**Comment by dreampiggy** [comment]
> Thank you!

Added in the README.md: https://github.com/SDWebImage/SDWebImage/commit/9cf8d125f522df0a7c3956f3828522ef2b6975fc
